src/lightning/pytorch/accelerators/hpu.py

```diff
@@ -0,0 +1,29 @@
+# Copyright The Lightning AI team.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from typing import Any
+
+from lightning.pytorch.accelerators.accelerator import Accelerator
+
+
+class HPUAccelerator(Accelerator):
+    """Accelerator for HPU devices."""
+
+    def __init__(self, *_: Any, **__: Any):
+        raise NotImplementedError(
```

Comment:
We have the graveyard for this. Lets use it instead of adding these classes

You can find examples by looking at the 1.9.x branch

Reply:
sure, could you pls check if I did it correctly, was not sure about what path and if setting class too...